src/nodes.coffee

```diff
@@ -3097,12 +3211,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
-
+  objectWithoutKeys: -> 'function(obj, props) { return Object.keys(obj).reduce(function(a,c) { return (![].includes.call(props, c)) && (a[c] = obj[c]), a; }, {}); }'
```

Comment:
This looks like an expensive implementation. What about
```coffee
objectWithoutKeys: "function(o, ks) {
    var res = {};
    for (var k in o) {
      indexOf.call(ks, k) < 0 && hasProp.call(o, k) && res[k] = o[k]
    }
    return res;
  }"
```
(you don't have to inline it manually, CS `"strings"` will do that for you. It's a couple characters shorter because I used shorter identifiers, but you can use longer ones.